_docs_operate/modules.md

```diff
@@ -87,6 +87,11 @@ For this, the Connector supports the configuration of webhooks which are called
 
 Keep in mind that you need to synchronize the state of the Connector with the Backbone in order to receive events. The `sync` Module automates this, but you can also do this manually by calling the `/api/v2/Account/Sync` route.
 
+### Sever sent events <a href="{% link _docs_operate/configuration.md %}#sse"><i class="fas fa-fw fa-cog"/></a> {#sse}
+
+The server sent events module allows the connector to open an one way connection from the backbone to the connector.
+This allows the backbone to communicate needed syncs to the connector and ditch the sync module that polls every x seconds.
```

Comment:
I'd rephrase: "...and ditch**es** the `sync` Module, which polls changes from the Backbone at regular time intervals.".

Reply:
I completely rephrased it into two sentences.